data/net/minecraft/client/gui/components/StateSwitchingButton.mapping

```diff
@@ -1,4 +1,10 @@
 CLASS net/minecraft/client/gui/components/StateSwitchingButton
+	METHOD <init> (IIIIZ)V
+		ARG 1 x
+		ARG 2 y
+		ARG 3 width
+		ARG 4 heigth
+		ARG 5 isStateTriggered
```

Comment:
I know we have the convention of following the field name for parameters if directly assigned, but I'd like to note that this could also potentially be `initialState`. For now though, we should follow convention.